py/parse_and_notify.py

```diff
@@ -218,15 +276,26 @@ def main():
             f"{dry_run}Topicctl ran apply on topic {topic_content.name} "
             f"in region {SENTRY_REGION}"
         )
-        text = topic_content.render_table()
-        if len(text) > 3950:
-            text = (
-                "Changes exceed 4000 character limit, "
-                "check topicctl logs for details on changes"
+        dd_table = topic_content.render_table(Destinations.DATADOG)
+        slack_table = topic_content.render_table(Destinations.SLACK)
+
+        too_long_message = (
+            "Changes exceed character limit, "
+            "check topicctl logs for more details on changes"
```

Comment:
Just add `...` 
So you do not waste a lot of characters if you are hitting the max length